builder/oracle/oci/config.go

```diff
@@ -195,6 +201,30 @@ func NewConfig(raws ...interface{}) (*Config, error) {
 		}
 	}
 
+	// Optional UserData config
```

Comment:
would be great if we had a utility function that took a `data` and `file_path` as input and returned the correct data. I feel like we use this pattern a lot

Reply:
Probably a good optimization, but at this point it'd require backwards incompatibilities for what is a really small improvement in UI

Reply:
Oh you mean just a helper function to do the parsing, not to change the template vars. Yeah, that seems like something that would make the code a little DRYer.